File: py/005.py
def gcd(a, b):
	if b == 0: return a
	return gcd(b, a % b)

def lcm(a, b):
	return a * b / gcd(a, b)

n = 1
for i in range(1, 20):
	n = lcm(n, i)
print(n)

File: c/005.c
#include <stdio.h>

long gcd(long a, long b) {
	if (b == 0) return a;
	return gcd(b, a % b);
}

long lcm(long a, long b) {
	return a * b / gcd(a, b);
}

int main() {
	long n = 1;
	for (int i = 1; i <= 20; i++) {
		n = lcm(n, i);
	}
	printf("%ld\n", n);
}
